app/src/main/java/org/sopt/dateroad/data/repositoryimpl/AuthRepositoryImpl.kt

```diff
@@ -0,0 +1,13 @@
+package org.sopt.dateroad.data.repositoryimpl
+
+import javax.inject.Inject
+import org.sopt.dateroad.data.dataremote.datasource.AuthRemoteDataSource
+import org.sopt.dateroad.domain.repository.AuthRepository
+
+class AuthRepositoryImpl @Inject constructor(
+    private val authRemoteDataSource: AuthRemoteDataSource
+) : AuthRepository {
+    override suspend fun getNicknameCheck(name: String) {
+        return authRemoteDataSource.getNicknameCheck(name = name)
+    }
```

Comment:
```suggestion
    override suspend fun getNicknameCheck(name: String) = authRemoteDataSource.getNicknameCheck(name = name)
```

리턴값만 존재하는 경우 = 으로 작성해줍시당

Reply:
runCatching 사용해주세요